senders/beanstlakd_sender.go

```diff
@@ -0,0 +1,60 @@
+package senders
+
+import (
+  "encoding/json"
+  "fmt"
+  "time"
+
+  interfaces "github.com/shoplineapp/captin/interfaces"
+  models "github.com/shoplineapp/captin/models"
+  beanstalk "github.com/beanstalkd/go-beanstalk"
+  log "github.com/sirupsen/logrus"
+)
+
+var bLogger = log.WithFields(log.Fields{"class": "BeanstalkdSender"})
+
+// BeanstalkdSender - Send Event to beanstalkd
+type BeanstalkdSender struct {
+  interfaces.EventSenderInterface
+}
+
+// SendEvent - #BeanstalkdSender SendEvent
+func (c *BeanstalkdSender) SendEvent(e models.IncomingEvent, d models.Destination) error {
+  host := fmt.Sprintf("%v", e.Control["beanstalkd_host"])
```

Comment:
Can use defer, recover and panic for error handling 🤔
https://blog.golang.org/defer-panic-and-recover 

```go
defer func() {
  if err := recover(); err != nil {
    bLogger.WithFields(log.Fields{
      "error": err,
    }).Error(err)
  }
  return err
}()
```